src/main/java/com/nvidia/spark/rapids/jni/SparkResourceAdaptor.java

```diff
@@ -56,9 +72,10 @@ public SparkResourceAdaptor(RmmEventHandlerResourceAdaptor<RmmDeviceMemoryResour
   public SparkResourceAdaptor(RmmEventHandlerResourceAdaptor<RmmDeviceMemoryResource> wrapped,
       String logLoc) {
     super(wrapped);
-    watchDog = new Thread(() -> {
+    // We are going to exit, so ignore the exception
```

Comment:
nit: this comment seems misplaced - it makes more sense in line 83 where it is I guess copied from

Reply:
IMO the other instance of this comment https://github.com/NVIDIA/spark-rapids-jni/blob/e943b86be513593675bbd56f3c3c461c17030daa/src/main/java/com/nvidia/spark/rapids/jni/SparkResourceAdaptor.java#L82 is not correct either. Ensuring a normal exit on InterruptedException is handling it rather than ignoring it.

Reply:
@gerashegalov I agree it is not a great comment, but I am going to leave it because I didn't touch that line in this PR, and because your approved the PR.

Reply:
I did not expect a fix, just could not help but make a pedantic comment :) @revans2